bin/setup

```diff
@@ -22,8 +22,17 @@ function setup_package {
   fi
 }
 
-setup_package "overmind"
-setup_package "yarn"
+function setup_package {
+  output=$(brew ls --versions $1)
+  if [ -n $output ]; then
+    install_package $1
+  else
+    echo "$1 installed, continuing..."
+  fi
+}
+
+setup_cli_tool "overmind"
+setup_cli_tool "yarn"
```

Comment:
Instead of splitting this into two functions, could we do an or, or a nested if or else if?